src/model/currency.java

```diff
@@ -0,0 +1,16 @@
+package model;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import java.sql.Date;
+
+@Data
+@AllArgsConstructor
+public class currency {
```

Comment:
C2. Every java class must Start by a capital letter (use Currency)
The file must be renamed too (to Currency)